### PR TITLE
update notes on setting up server

### PR DESCRIPTION
**Pull request by KwadroNaut**
postgres section

### CODE REVIEW

File: docs/setup.rst [hunk 1]
```diff
@@ -87,30 +87,40 @@ Setting up a Provider
...
-          psql --username=provider --password -d katzenpost -f create_database-postgresql.sql
+          # set some passwords for your new user
+          psql
+          postgres=# ALTER USER provider WITH PASSWORD 'secrətp0stgre5sy';
 
-          # start katzenpost server...
+          # test if you can connect
+          psql -U provider -h 127.0.0.1 katzenpost
+
+          # If all goes fine, it's time to load th SQL, that script lives in
```
Comment (Yawning) [inline docs/setup.rst]: "time to load th SQL"

**Comment by KwadroNaut** [comment]
Thanks for noticing my mistakes, updated.

**Comment by Yawning** [comment]
Hmm.  I'm not sure if the BoltDB or PostgreSQL backends should be recommended by default.  In theory the PostgreSQL backend will scale a lot better, but in order to see any benefit, this will at a minimum require:

 * Increasing the provider worker concurrency to allow for parallelism. (Via an entirely undocumented Debug config option).
 * Tuning the Postgresql config.  If the Debian package's configuration is anything like the upstream one, it is super conservative with regards to how much memory the server can use (among other things), and will require adjustment on a per-installation basis.

Without doing both of those things, the experience is probably going to be worse.  For smaller installations (or for testing), it probably makes sense to stick with BoltDB for now, despite the fact that I just spend the last week adding support for using an RDBMS.

**Comment by KwadroNaut** [comment]
I agree that the whole PostgreSQL stuff is very low priority now. However the current stuff that's online is draft-quality with errors. ([see](https://github.com/katzenpost/website/blob/e6f49370e91f5b4d234881b6c923a38486b78053/docs/setup.rst) ), this merely tries to take those out. It's currently not even mentioned in the TOC. We can add at the top a short line, something like you said that tuning is to be expected, it might be easier for integration the user db but that the other stuff works fine as well for testbeds? Or would you want to move it to another section of the documentation?

**Comment by moba** [comment]
It's nice to already have it documented, thanks! But I agree that we should clarify that this is an optional step and not required if you are fine with BoltDB.

**Pull request merged**